hbase-client/src/main/java/org/apache/hadoop/hbase/client/RpcConnectionRegistry.java

```diff
@@ -49,7 +49,7 @@
 public class RpcConnectionRegistry extends AbstractRpcBasedConnectionRegistry {
 
   /** Configuration key that controls the fan out of requests **/
-  public static final String HEDGED_REQS_FANOUT_KEY = "hbase.client.rpc_registry.hedged.fanout";
+  public static final String HEDGED_REQS_FANOUT_KEY = "hbase.client.bootstrap.hedged.fanout";
```

Comment:
hbase.client.rpc_registry.hedged.fanout is a new config recently added. It is committed to a branch only currently? So it is fine changing the name? Does the config get doc'd in release notes?

Reply:
Yes, no actual release yet, this is a missing part in the previous commit, @bharathv suggest that we do not use 'rpc_registry', just use 'bootstrap', but I forgot to change this one.
No release note yet. We could mention it in the release note for this issue. Or better describe it in the ref guide? Usually you do not need to change this value.